src/parent/parent.js

```diff
@@ -316,6 +317,7 @@ export function parentComponent<P, X, C>({
   let childComponent: ?ChildExportsType<P>;
   let currentChildDomain: ?string;
   let currentContainer: HTMLElement | void;
+  let isSecondRenderFinished: boolean = false;
```

Comment:
Can we rename this to render? I think the existing naming convention in zoid is prerender (phase 1) and render (phase 2) 
```suggestion
  let isRenderFinished: boolean = false;
```